tests/unit/ssh/test_ssh.py

```diff
@@ -14,10 +14,12 @@
 
 # Import Salt libs
 import salt.config
+import salt.utils
 from salt.client import ssh
 
 
 @skipIf(NO_MOCK, NO_MOCK_REASON)
+@skipIf(not salt.utils.which('ssh'), "No ssh binary found in path")
```

Comment:
This should use the new utils location `salt.utils.path.which` (and the import should be changed as well). Otherwise 👍

Reply:
Fixed. Switching between the 2017.7 and 2018.3 branch made me do this mistake.